core/src/main/java/hudson/cli/DisablePluginCommand.java

```diff
@@ -119,14 +119,10 @@ private void printIndented(int indent, String format, String... arguments) {
         if (indent == 0) {
             stdout.format(format + "%n", (Object[]) arguments);
         } else {
-            String f = "%" + indent + "s" + format + "%n";
             String[] newArgs = new String[arguments.length + 1];
-            newArgs[0] = " ";
-            int i = 1;
-            for (String arg: arguments) {
-                newArgs[i] = arg;
-                i++;
-            }
+            System.arraycopy(arguments, 0, newArgs, 1, arguments.length);
```

Comment:
WYT @varyvol ? :wink:

Reply:
Better, but I think you removed too much (the assigment to newArgs[0]).